Lesson31/Kadir/my-app/src/components/AddButton/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import "./styles.css";
+
+export const AddButton = ({ key, data, setData}) => {
+  const [text, setText] = useState("");
+  const [newItem, setNewItem] = useState(); // Define a state to store the newItem
```

Comment:
These state is not used, text and setText are used instead. 'newItem' and 'setNewItem' names are better, so maybe update the names :)